wiki/Tournaments/GBC/GBC_2023_Spring/en.md

```diff
@@ -0,0 +1,207 @@
+| Designer | ::{ flag=CN }:: [\[GB\]Voidmoyu](https://osu.ppy.sh/users/16573996), ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) |
+| Referee | ::{ flag=CN }:: [\[GB\]Luoxuan0327](https://osu.ppy.sh/users/8586018), ::{ flag=CN }:: [\[GB\]Natro](https://osu.ppy.sh/users/27233308), ::{ flag=CN }:: [Hylotl](https://osu.ppy.sh/users/18270260), ::{ flag=CN }:: [\[GB\]V1do](https://osu.ppy.sh/users/17527968), ::{ flag=CN }:: [Lucky Clover](https://osu.ppy.sh/users/24989144), ::{ flag=CN }:: [YuEast 2018](https://osu.ppy.sh/users/13953619), ::{ flag=CN }:: [\[GB\]sherweifa](https://osu.ppy.sh/users/17457562), ::{ flag=CN }:: [\[Crz\]Alleyne](https://osu.ppy.sh/users/11279273), ::{ flag=CN }:: [\[GB\]Akamite](https://osu.ppy.sh/users/13418334), ::{ falg=CN }:: [alivn](https://osu.ppy.sh/users/15807665) |
+| Statistician & wiki editor | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [\[Crz\]Alleyne](https://osu.ppy.sh/users/11279273), ::{ flag=CN }:: [\[GB\]Natro](https://osu.ppy.sh/users/27233308), ::{ flag=CN }:: [FAIRYBYUN](https://osu.ppy.sh/users/17575228) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1730051)
+- Livestream
+  - [Bilibili stream](https://live.bilibili.com/22545296) (::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407))
+  - [Bilibili stream](https://live.bilibili.com/10183) (::{ flag=CN }:: [aiyulu](https://osu.ppy.sh/users/189617))
+  - [Bilibili stream](https://live.bilibili.com/6605693) (::{ flag=CN }:: [\[Crz\]Alleyne](https://osu.ppy.sh/users/11279273))
+- [GBC QQ Group](https://jq.qq.com/?_wv=1027&k=zqBeTqMW)
+- [Information spreadsheet](https://docs.qq.com/sheet/DTXNmQXJ2VG9qbWhY?tab=uitssi)
+
+## Participants
+
+| Group | Player(s) |
+| :-: | :-- |
+| Group EX | *TBD* |
+| Group S | *TBD* |
+| Group A | *TBD* |
+| Group B | *TBD* |
+
+## Match results
+
+Due to the large number of players, the match results will be displayed in the [information spreadsheet](https://docs.qq.com/sheet/DTXNmQXJ2VG9qbWhY?tab=uitssi).
+
+## Ruleset
+
+### General
+
+1. The GB Cup 2023 Spring is a point-based weekly group tournament, played on the osu!mania game mode.
+   - Points earned by players in weekly matches are converted based on the total ranking of the mappool.
+   - The total ranking of the mappool is based on the sum of the rankings of each player in each map.
+2. Beatmap scoring is based on Score V2.
+3. The GB Cup 2023 Spring consists of **4 groups (EX, S, A, and B)**, with only **2 groups** playing each week.
+   - Each group has an overrun buffer of *100 pp*.
+   - If a player upgrades after the start of the tournament, 50% of the points will be kept.
+4. The maps for each round will be announced by the mappool selectors on the official stream, on the Sunday before the actual matches take place.
+   - Each mappool contains 7 beatmaps of varying types.
```

Comment:
this should be in the mappool information section

Reply:
duplicate of mappool info point 3
remove